### DISCUSSION TITLE
Need help with commands for video transmission.

### DISCUSSION THREAD

**Discussion by Jcwscience** [svpcom/wfb-ng#190] [Questions] [answered]
I am trying to connect a raspberry pi 4 to an Nvidia AGX Xavier, and my (sort of working) setup is this

on the drone 
```
libcamera-vid --inline -n -t 0 -b 5000000 --framerate 30 --width 1920 --height 1080 --level 4.2 --profile high --intra 15 -o - | gst-launch-1.0 -v fdsrc ! h264parse ! rtph264pay config-interval=1 pt=35 ! udpsink sync=false host=127.0.0.1 port=5602
```

and on the ground station

```
gst-launch-1.0 udpsrc port=5600 ! 'application/x-rtp,encoding-name=H264,payload=35' ! rtph264depay ! h264parse ! nvv4l2decoder enable-max-performance=1 ! nvvidconv ! 'video/x-raw(memory:NVMM),width=1920,height=1080 ! nvvidconv ! glupload ! glimagesink
```

Is there anything here that is redundant that can be trimmed out? And if I were to split the stream off to another program or maybe a v4l2loopback device, where in this mess would I find a useable h264 stream?

Thanks!

**Comment by svpcom** [comment, marked as answer]
You can set peer as multicast address `224.0.0.1`. But this will work only for different computers on the same lan (not for different programs on one computer). For different programs you can use gstreamer pipeline to mirror rtp stream to multiple destinations
